src/config.js

```diff
@@ -67,6 +67,11 @@ export default Object.assign(
      * required by the Sentry SDKs.
      */
     SENTRY_DSN: '',
+
+    /**
+     * URL of the web client
+     */
+    WEB_CLIENT: 'http://localhost:8095/',
```

Comment:
1. should not contain trailing `/`
2. should be named `WEB_CLIENT_URL`

Reply:
Removed slash, renamed to WEB_CLIENT_URL